packages/block-editor/README.md

```diff
@@ -484,6 +484,7 @@ _Properties_
 -   _alignWide_ `boolean`: Enable/Disable Wide/Full Alignments
 -   _availableLegacyWidgets_ `Array`: Array of objects representing the legacy widgets available.
 -   _imageEditing_ `boolean`: Image Editing settings set to false to disable.
+-   _imageCrossOrigin_ `[string]`: Optional value for requesting images with CORS, allowing them to be used in a `<canvas>` as if they were loaded from the current origin.
```

Comment:
Might be worth explaining what possible values there are.

Reply:
Oh, and is this restricted for use in images or can this also be used for other media in the future?

Reply:
Video elements seem to use the same attribute, I'd say it's potentially any media that can be used in a canvas:

> `crossorigin`
> This enumerated attribute indicates whether to use CORS to fetch the related video. CORS-enabled resources can be reused in the <canvas> element without being tainted. The allowed values are:

Maybe `mediaCrossOrigin` is best. We'll work on better description.

Reply:
Let's say the value should be one of:

- omitted (`undefined`)
- `""` or `"anonymous"`
- `"use-credentials"`